src/mcts/stoppers/factory.cc

```diff
@@ -292,13 +308,52 @@ std::unique_ptr<SearchStopper> LegacyTimeManager::CreateTimeManagementStopper(
   return std::make_unique<LegacyStopper>(deadline, &time_spared_ms_);
 }
 
+std::unique_ptr<SearchStopper>
+LegacyTimeManager::CreateAlphaZeroTimeManagementStopper(
+    const OptionsDict& options, const GoParams& params,
+    const Position& position) {
+  const bool is_black = position.IsBlackToMove();
+  const optional<int64_t>& time = (is_black ? params.btime : params.wtime);
```

Comment:
std::optional

Reply:
Thanks I had to install Toolset Visual Studio 2017 (v141), you must install the following components in Visual Studio 2019 Community edition to properly see and solve the build issue:
C++ ATL for v141 build tools (x86 & x64)
C++ MFC for v141 build tools (x86 & x64)
Done and solved